SDL_ttf.c

```diff
@@ -2985,13 +3101,10 @@ static int TTF_Size_Internal(TTF_Font *font,
         int y_offset  = hb_glyph_position[g].y_offset;
 #else
     /* Load each character and sum it's bounding box */
-    textlen = SDL_strlen(text);
```

Comment:
The local `size_t textlen;` should be removed from the top of the procedure, as well.